scripts/check-go-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+MINIMAL_VERSION=1.13
```

Comment:
Minimal version is 1.14 for this repository, because we use the `-modfile` argument for `go build`.

Reply:
Suggest the script take the version directly from go.mod to avoid duplication: 
```
MINVERSION=$(awk '/^go/ {print $2}' go.mod)
```

Reply:
done